src/beautifiers/csscomb.coffee

```diff
@@ -6,10 +6,10 @@ module.exports = class JSBeautify extends Beautifier
 
   options: {
     # TODO: Add support for options
-    CSS: false
-    LESS: false
-    Sass: false
-    SCSS: false
+    CSS: true
```

Comment:
Change:

```
    CSS: true
    LESS: true
    Sass: true
    SCSS: true
```

to:

```
    _:
      configPath: true
      predefinedConfig: true
    CSS: true
    LESS: true
    Sass: true
    SCSS: true
```

Similar to https://github.com/danielbayley/atom-beautify/blob/master/src/beautifiers/prettydiff.coffee#L6

This allows only the options that are supported (`configPath` and `predefinedConfig`) and puts that for the default (`_`).